src/keri/app/cli/commands/witness/start.py

```diff
@@ -54,13 +60,16 @@ def launch(args):
                alias=args.alias,
                bran=args.bran,
                tcp=int(args.tcp),
-               http=int(args.http))
+               http=int(args.http),
+               config_dir=args.config_dir,
+               config_file=args.config_file)
 
     logger.info("\n******* Ended Witness for %s listening: http/%s, tcp/%s"
                 ".******\n\n", args.name, args.http, args.tcp)
 
 
-def runWitness(name="witness", base="", alias="witness", bran="", tcp=5631, http=5632, expire=0.0):
+def runWitness(name="witness", base="", alias="witness", bran="", tcp=5631, http=5632, expire=0.0,
+               config_dir="", config_file=""):
```

Comment:
In KERIpy, camelcase is standard for parameters.  Can you update these two?

Reply:
Yes, just pushed the update. I should get a pre-commit hook set up with a code style check.